peripherals/sdio/sdio_f446.yaml

```diff
@@ -0,0 +1,258 @@
+_modify:
+  SDMMC:   # renaming block to be RM0390-compliant
+    name: SDIO
+    description: 'Secure digital input/output SD/SDIO MMC card host interface'
+    groupName: SDIO
+    addressBlock:
+      size: "0x400"     
+
+SDIO:
+  _strip: SDMMC_
+  _delete:
+    - '*ACKTIMER'
+    - '*IDMA*'
+    - '*VER'
+    - '*ID'
+  _modify:
+    SDMMC_POWER:
+      description: 'SDIO power control register'
+    SDMMC_CLKCR:
+      description: 'CLKCR register controls the SDIO_CK output clock.'
+    SDMMC_CMDR:
+      name: "SDMMC_CMD"  # NOTE: something weird happens on leaving just 'CMD' -- 'MD' -- ?
+    SDMMC_RESP4R:
+      addressOffset: "0x20"
+      name: RESP4
+    SDMMC_RESP3R:
+      addressOffset: "0x1C"
+      name: RESP3
+    SDMMC_RESP2R:
+      addressOffset: "0x18"
+      name: RESP2
+    SDMMC_RESP1R:
+      addressOffset: "0x14"
+      name: RESP1
+    SDMMC_RESPCMD:
+      addressOffset: "0x10"
+    SDMMC_ARGR:
+      name: ARG
+    SDMMC_RESPCMDR:
+      name: RESPCMD
+    SDMMC_DLENR:
+      name: "SDMMC_DLEN"
+    SDMMC_DCNTR:
+      name: 'SDMMC_DCOUNT'
+    SDMMC_STAR:
+      name: SDMMC_STA
+    SDMMC_MASKR:
+      name: SDMMC_MASK
+    SDMMC_FIFOR:
+      addressOffset: "0x80" # acc. to CMSIS headers (??)
```

Comment:
0x80 seems correct in RM0390 too, where SDIO_FIFO is listed at address offset 0x80